go.mod

```diff
@@ -1,17 +1,21 @@
 module github.com/lazyledger/rsmt2d
 
-go 1.14
+go 1.15
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/kr/text v0.2.0 // indirect
```

Comment:
Can we at least understand what caused this to be imported? If it is gonum, we will remove gonum with #22.

Reply:
For posterity: this is dependency of `yaml` v3, which is a dependency of `testify`